.vuepress/config.js

```diff
@@ -7,118 +13,119 @@ module.exports = {
-        items: [{
-            text: 'Home',
-            link: '/wiki/'
-          },
-          {
-            text: 'Troubleshooting',
-            link: '/wiki/troubleshooting'
-          },
-          {
-            text: 'Fixing the Microsoft Store',
-            link: '/wiki/fixing-microsoft-store'
-          },
-          {
-            text: 'Installing Windows',
-            link: '/wiki/installing-windows'
-          },
-          {
-            text: "Joining and Leaving the Windows Insider Program",
-            link: "/wiki/windows-insiders"
-          },
-          {
-            text: 'Reinstalling Graphics Drivers',
-            link: '/wiki/reinstalling-gpu-drivers'
-          },
-          {
-            text: 'Using the Media Creation Tool',
-            link: '/wiki/using-the-media-creation-tool'
-          },
-          {
-            text: 'Common Windows Misconceptions',
-            link: '/wiki/common-misconceptions'
-          },
-          {
-            text: 'Backing Up Data',
-            link: '/wiki/backup'
-          },
-          {
-            text: 'The Windows Feature Store',
-            link: '/wiki/windows-feature-store'
-          }
-        ]
-      },
-      {
-        text: 'About',
-        items: [{
-            text: 'Join Discord server',
-            link: 'https://aka.ms/community-discord'
-          },
-          {
-            text: 'Bots',
-            link: '/about/bots'
-          },
-          {
-            text: 'Contributors',
-            link: '/about/contributors'
-          },
-          {
-            text: 'Moderation team',
-            link: '/about/moderators'
-          },
-          {
-            text: 'Community reviews',
-            link: 'https://disboard.org/server/150662382874525696'
-          }
-        ]
+    navbar: [{
+      text: 'Member Information',
+      link: '/member/'
+      // items: [
+      //   { text: 'Rules', link: '/member/#rules' },
+      //   { text: 'Roles', link: '/member/#roles' },
+      //   { text: 'Moderator Help', link: '/member/#moderator-help' },
+      //   { text: 'Channels', link: '/member/#channels' },
+      //   { text: 'Available commands', link: '/member/#commands' },
+      //   { text: 'FAQ', link: '/member/#frequently-asked-questions' },
+      //   { text: 'Similar communities', link: '/member/#similar-communities' },
+      // ]
+    },
+    {
+      text: 'Moderator Resources',
+      children: [{
+        text: 'Moderator Documentation',
+        link: '/mod/docs/'
+      },
+      {
+        text: 'Apply for mod position',
+        link: 'https://apply.msft.chat'
+      },
+      {
+        text: 'Server Analytics',
+        link: 'https://statbot.net/dashboard/150662382874525696'
+      },
+      {
+        text: 'Server Reviews',
+        link: 'https://disboard.org/server/reviews/150662382874525696'
+      }
+      ]
+    },
+    {
+      text: 'Wiki',
+      children: [{
+        text: 'Home',
+        link: '/wiki/'
```

Comment:
This will stop Home from being highlighted on all pages
```suggestion
        link: '/wiki/',
        activeMatch: '/wiki/$'
```